docs/develop/common-coding-issues.md

```diff
@@ -1,14 +1,20 @@
 ---
 title: Common coding issues and unexpected platform behaviors
 description: 'A list of Office JavaScript API platform issues frequently encountered by developers.'
-ms.date: 10/29/2019
+ms.date: 10/30/2019
 localization_priority: Normal
 ---
 
 # Common coding issues and unexpected platform behaviors
 
 This article highlights aspects of the Office JavaScript API that may result in unexpected behavior or require specific coding patterns to achieve the desired outcome. If you encounter an issue that belongs in this list, please let us know by using the feedback form at the bottom of the article.
 
+## Common and Outlook APIs are not promise-based
+
+The [Common APIs](/javascript/api/office) (those that are not tied to a particular host) and [Outlook APIs](/javascript/api/outlook) use a callback-based programming model. Interacting with the underlying Office document requires an asynchronous read or write call that includes a callback to be ran when the operation completes. For an example of this pattern, see [AppointmentCompose.getAttachmentAsync](/javascript/api/office/office.file?view=common-js#getsliceasync-sliceindex--callback-).
```

Comment:
The URL target (bookmark `#getsliceasync-sliceindex--callback-`) specified at the end of this paragraph for the link `AppointmentCompose.getAttachmentAsync` doesn't point to the right place in the reference docs.